modules/tfc-agent-mig-container-vm/main.tf

```diff
@@ -159,7 +159,7 @@ module "mig_template" {
   version            = "~> 7.0"
   region             = var.region
   project_id         = var.project_id
-  network            = local.network_name
+  network            = local.network_name == "null" ? null : local.network_name
```

Comment:
This is kind of weird, using the string `null` I mean. Why not check if it's really `null`?

I don't know exactly how to handle it, but here are some ideas :-D:

- [`coalesce` function](https://developer.hashicorp.com/terraform/language/functions/coalesce)
- [nullable variables](https://developer.hashicorp.com/terraform/language/values/variables#disallowing-null-input-values)
- [`try` function](https://developer.hashicorp.com/terraform/language/functions/try)

Have you tried them?

Reply:
Thanks @fmadrids ! Let me try them :)

Reply:
Working. Thank you @fmadrids !